src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -109,4 +111,22 @@ void move(@TempDir final Path tmp) {
         storage.move(source, destination);
         MatcherAssert.assertThat(storage.value(destination), Matchers.equalTo(data));
     }
+
+    @Test
+    void list(@TempDir final Path tmp) {
```

Comment:
@olegmoz let's also test some negative cases when no objects found.

Reply:
@iakunin nice point! Added the test and changed behavior of list() method in this case, so it doesn't crash, but returns empty collections. Think it's better. Thank you!

Reply:
@olegmoz thank you for your responsiveness =)